src/Shared/DC/ZRDB/RDB.py

```diff
@@ -34,8 +32,7 @@ def parse_text(s):
     return '\\'.join(r)
 
 
-if six.PY3:
-    long = int
+long = int
```

Comment:
I'd appreciate replacing `long` with `int` in this module.

Reply:
Done